lib/mobility/plugins/active_record.rb

```diff
@@ -4,6 +4,7 @@
 require_relative "./active_record/cache"
 require_relative "./active_record/query"
 require_relative "./active_record/uniqueness_validation"
+require_relative "./active_record/original_column"
```

Comment:
We could maybe debate whether this is a good name.

Reply:
I was thinking the name could include the word "fallback" or "fallthrough" like with `fallthrough_accessors`, since the plugin only interferes if certain conditions are met. Maybe `fallthrough_original_column`, `original_column_fallthrough`?

Reply:
Yeah hmm still thinking about it :thinking: